k8sutils/pkg/describe/odigos.go

```diff
@@ -22,6 +22,11 @@ type clusterCollectorResources struct {
 	LatestRevisionPods *corev1.PodList
 }
 
+type nodeCollectorResources struct {
+	CollectorsGroup *odigosv1.CollectorsGroup
+	DaemonSet       *appsv1.DaemonSet
```

Comment:
I think we should also include the status for the node collector pods similar to what was done for the cluster collector. In this case we will actually have pod in each node so reporting their status is useful (If a some nodes are not in good shape).
Or at least to write `x/y node collectors are healthy`

Reply:
I added this info based on the ds status fields.
It doesn't read as nicely, but it's suppose to give indication if something is not running.
See the screenshot in the PR description